types/mori/tsconfig.json

```diff
@@ -0,0 +1,21 @@
+{
+    "compilerOptions": {
+        "module": "commonjs",
+        "noEmit": true,
+        "forceConsistentCasingInFileNames": true,
+        "baseUrl": "../",
+        "typeRoots": ["../"],
+        "types": [],
+        "strictFunctionTypes": false,
+        "lib": [
+            "es6"
+        ],
+        "noImplicitAny": false,
```

Comment:
noImplicitAny should be true. Are the errors you're seeing from it only in the tests? Everything in the declaration file seems typed.

Reply:
I can fix these. Plan to work on it tomorrow.

Reply:
Done.